src/Xamarin.Android.Build.Tasks/Utilities/AndroidResource.cs

```diff
@@ -202,6 +202,8 @@ private static bool TryFixResAuto (XAttribute attr, Dictionary<string, string> a
 		{
 			if (attr.Name.Namespace != res_auto)
 				return false;
+			if (!attr.Name.LocalName.StartsWith ("@")) // only lowercase if we reference another resource.
```

Comment:
Should this actually be looking at `attr.Value`? I think the value of the attribute will have the `@` symbol?

Reply:
🤦‍♂